src/h5cli/cli.py

```diff
@@ -104,6 +104,23 @@ def do_bang(self, args, opts=None):
     def do_clear(self, args):
         os.system('clear')
 
+    @options([
+        make_option('-n', '--number',action='store_true', help='print number of elements, rather than full shape'),
+        make_option('-m', '--maxshape',action='store_true', help='print maximum shape, rather than current shape')
```

Comment:
From the logic in the function it seems like these two options are mutually exclusive. Is there a way to specify that these options can't go together?